internal/coremsgs/en_config_descriptions.go

```diff
@@ -259,17 +259,20 @@ var (
 	ConfigMetricsReadTimeout  = ffc("config.metrics.readTimeout", "The maximum time to wait when reading from an HTTP connection", i18n.TimeDurationType)
 	ConfigMetricsWriteTimeout = ffc("config.metrics.writeTimeout", "The maximum time to wait when writing to an HTTP connection", i18n.TimeDurationType)
 
```

Comment:
I think there are some "fireflyContracts" fields for both ethereum and fabric that can now be deleted from this file.